doc/api/deprecations.md

```diff
@@ -2786,6 +2786,19 @@ These properties are now available within the standard `detail` property
 of the `PerformanceEntry` object. The existing accessors have been
 deprecated and should no longer be used.
 
+### DEP0153: Non boolean value for `verbatim` DNS lookup option
```

Comment:
I'm not sure — this string may appear elsewhere.
```suggestion
### DEP0153: Non-boolean value for `verbatim` DNS lookup option
```

Reply:
If we're using _non-boolean_, let's definitely hyphenate like that. Another option, though, would be to reword to say what it must be rather than what it can't be:

```suggestion
### DEP0153: DNS lookup option `verbatim` must be boolean

Reply:
Actually, maybe not on that rewording idea....